lib/data/dao/note_helper.dart

```diff
@@ -13,15 +13,23 @@ class NoteHelper extends DatabaseAccessor<AppDatabase> with _$NoteHelperMixin {
     switch (mode) {
       case ReturnMode.NORMAL:
         return (select(notes)
-              ..where((table) => table.archived.not() & table.deleted.not()))
+              ..where((table) => table.archived.not() & table.deleted.not() & table.id.contains("-synced").not()))
```

Comment:
you may prefer to use endsWith here instead of contains

Reply:
You are totally right. This is really useful feedback, thanks!

Reply:
Apparently the type of id is GeneratedTextColumn, which doesnt have all the methods a string has. Looks like it will have to keep using contains, even though endsWith would be better.

Reply:
i can look into it dw